### DISCUSSION TITLE
How neovim configures multiple split-screen windows to use the same status bar

### DISCUSSION THREAD

**Discussion by Sammorn** [rebelot/heirline.nvim#202] [General]
How neovim configures multiple split-screen windows to use the same status bar

**Comment by rebelot** [comment]
`set laststatus=2` and something like `require'heirline'.setup{statusline = {provider='my unique statusline'}}`

**Comment by Sammorn** [comment]
It didn't work out as intended, and each split-screen window was still using its own status line
<img width="831" alt="1718811496103" src="https://github.com/rebelot/heirline.nvim/assets/91642847/b6adac36-6bad-4945-82af-e7cce2c11a67">

**Comment by rebelot** [comment]
Sorry I didn't understand properly.. set laststatus option to 3

**Comment by Sammorn** [comment]
It worked this time. Thanks.
